var/spack/repos/builtin/packages/cbtf-krell/package.py

```diff
@@ -83,29 +89,53 @@ class CbtfKrell(CMakePackage):
     # Dependencies for cbtf-krell
     depends_on("cmake@3.0.2:", type='build')
 
-    # For binutils service
-    depends_on("binutils")
+    # For binutils
+    depends_on("binutils", when='@develop')
+    depends_on("binutils@2.29.1", when='@1.9.1.0')
```

Comment:
Does this directly rely on binutils? See http://spack.readthedocs.io/en/latest/getting_started.html#binutils

Reply:
Yes, we use the libiberty library and libbfd to resolve symbols.

Reply:
@adamjstewart I think I missed the comment about all the duplication.   Is this referring to the same package having two depends_on statements?   My co-worked wants a release path and development path through the spack build process.   So, we created a release branch 1.9.1.0 for the cbtf packages and 2.3.1.3 for the openspeedshop packages.   Is there a better way to do what we are attempting?
Thanks
Jim G